src/Security/Security/Cmdlets/Pricings/SetPricings.cs

```diff
@@ -37,6 +39,9 @@ public class SetPricings : SecurityCenterCmdletBase
         [Parameter(ParameterSetName = ParameterSetNames.SubscriptionLevelResource, Mandatory = false, HelpMessage = ParameterHelpMessages.SubPlan)]
         public string SubPlan { get; set; }
 
+        [Parameter(ParameterSetName = ParameterSetNames.SubscriptionLevelResource, Mandatory = false, HelpMessage = ParameterHelpMessages.Extensions)]
+        public string Extensions { get; set; }
```

Comment:
- please use 'Extension' instead of 'Extensions'
- please re-generate help markdown for new parameter

Reply:
Sure